lib/tool_shed/util/shed_util_common.py

```diff
@@ -512,6 +512,11 @@ def get_repository_for_dependency_relationship( app, tool_shed, name, owner, cha
                                                owner=owner,
                                                changeset_revision=changeset_revision )
     if not repository:
+        tool_shed_url = common_util.get_tool_shed_url_from_tool_shed_registry( app, tool_shed )
+        repository_clone_url = os.path.join( tool_shed_url, 'repos', owner, name )
+        repo_info_tuple = (None, repository_clone_url, changeset_revision, None, owner, None, None)
+        repository, pcr = repository_was_previously_installed( app, tool_shed_url, name, repo_info_tuple )
```

Comment:
@gregvonkuster we have dockerized TS tests running at https://jenkins.galaxyproject.org/job/docker-toolshed/ but currently 12 are failing because of lack of maintenance (that is why they are not plugged in the PRs like the others)

I also think that things are not going away soon. Not if we want to stay reproducible.